docs/team/markusyeo.md

```diff
@@ -9,35 +9,37 @@ Dengue Hotspot Tracker is a desktop app for managing dengue cases. The user inte
 
 Given below are my contributions to the project.
 
+* Given below are my contributions to the project.
+
+* **New Feature**: Changed default file storage format to be CSV
+  * Justification: As we deal with tabular data, we would like users to be able to easily view the data entered into `DengueHotspotTracker` offline.
+  Additionally NEA personnel are more likely to be familiar with CSV than json!
+  * Highlight: It was challenging to write to CSV as it required exploring a new package `OpenCSV` this was nonetheless a very fruitful experience in exploring a new codebase and adding it into our project. 
+  I had to add a new `CsvUtil` file to handle all the reading and writing from CSV.
+* **New Feature**: Added the ability to import and export data as a csv file
+  * Justification: This feature allows users to edit csv files to be able to import into `DengueHotspotTracker` and export from.
```

Comment:
```suggestion
  * Justification: This feature allows users to export, edit, and import csv files into `DengueHotspotTracker`. 
```
NGL, this doesn't work very well ("export [...] into"?), but I think it gets the point across :) Something like "export csv files from and import csv files into" would've been a lot clunkier.